en-US/firefox/windows-64-bit.lang

```diff
@@ -0,0 +1,55 @@
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/windows-64-bit/
+
+
+# HTML page title.
+;Firefox for Windows 64-bit
+Firefox for Windows 64-bit
+
+
+# HTML page description.
+;Users on 64-bit Windows who download Firefox can get our 64-bit version by default. That means you get a more secure version of Firefox.
+Users on 64-bit Windows who download Firefox can get our 64-bit version by default. That means you get a more secure version of Firefox.
+
+
+;64-bit
+64-bit
+
+
+;A more secure Firefox.
+A more secure Firefox.
+
+
+;Users on 64-bit Windows who download Firefox can get our 64-bit version by default. That means you get a more secure version of Firefox, one that also <a href="%(crashes)s">crashes a whole lot less</a>. How much less? In our tests so far, 64-bit Firefox reduced crashes by 39%% on machines with 4GB of RAM or more.
+Users on 64-bit Windows who download Firefox can get our 64-bit version by default. That means you get a more secure version of Firefox, one that also <a href="%(crashes)s">crashes a whole lot less</a>. How much less? In our tests so far, 64-bit Firefox reduced crashes by 39%% on machines with 4GB of RAM or more.
+
+
+;What’s the difference between 32-bit and 64-bit?
+What’s the difference between 32-bit and 64-bit?
+
+
+;Here’s the key thing to know: 64-bit applications can access more memory and are less likely to crash than 32-bit applications. Also, with the jump from 32 to 64 bits, a security feature called <a href=%(ASLR)s>Address Space Layout Randomization (ASLR)</a> works better to protect you from attackers. Linux and macOS users, fret not, you already enjoy a Firefox that’s optimized for 64-bit.
```

Comment:
Do we want to add a comment on `fret not`?

Reply:
Added a comment thanks